README.md

```diff
@@ -318,7 +339,7 @@ install this earlier, you can do so with the command:
 sudo apt-get install node-uglify
 ```
 
-## FRDM-K64F Platform
+## FRDM K64F Platform
```

Comment:
The dash is [how they identify it](https://developer.mbed.org/platforms/FRDM-K64F/).